tests/e2e/api/server.js

```diff
@@ -42,7 +42,7 @@ describe('server', () => {
           if (res.headers['content-type'] === 'application/json' && typeof body === 'string') {
             try {
               body = JSON.parse(body);
-            } catch (err) {}
+            } catch (err) {} /* eslint-disable-line no-empty */
```

Comment:
Instead of disabling this warning, we should add a comment as to why we're ignoring it. This is both better documentation and means we don't have to disable ESLint here

Reply:
I edited the code to log the error inside `catch (err) {}` you know what happened? The test failed and that's why I disabled `eslint` on that line

Reply:
According to [the docs](https://eslint.org/docs/rules/no-empty) you can just add a comment inside the block for why you're ignoring the error. I think that's the right approach here.